src/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
+import { Provider, compose } from 'react-redux';
```

Comment:
Oh sorry, it is when connecting to Redux dev tools:
`const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;` 

I copied that from another project. I do no use compose yet and that is why i removed it. I missed out on that it is used on that line.